client/components/navbar/navbar.service.js

```diff
@@ -1,226 +1,481 @@
+        const frenchHomeGuide = _.get(this.URLS.guides, "home.FR");
+        if (cloudGuide) {
+            assistanceMenu.push({
+                title: this.$translate.instant("common_menu_support_guide"),
+                url: cloudGuide,
+                isExternal: true
+            });
+        } else if (homeGuide) {
+            assistanceMenu.push({
+                title: this.$translate.instant("common_menu_support_all_guides"),
+                url: homeGuide,
+                isExternal: true
+            });
+        } else if (frenchHomeGuide) {
+            assistanceMenu.push({
+                title: this.$translate.instant("common_menu_support_all_guides"),
+                url: frenchHomeGuide,
+                isExternal: true
+            });
+        }
+
+        // New ticket
+        assistanceMenu.push({
+            title: this.$translate.instant("common_menu_support_new_ticket"),
+            click: (callback) => {
+                if (!this.otrsPopupService.isLoaded()) {
+                    this.otrsPopupService.init();
+                } else {
+                    this.otrsPopupService.toggle();
+                }
+
+                if (typeof callback === "function") {
+                    callback();
+                }
+            }
+        });
+
+        // Tickets list
+        assistanceMenu.push({
+            title: this.$translate.instant("common_menu_support_list_ticket"),
+            url: "#/support"
+        });
+
+        // Telephony (External)
+        if (this.TARGET !== "US") {
+            assistanceMenu.push({
+                title: this.$translate.instant("common_menu_support_telephony_contact"),
+                url: this.URLS.support_contact[locale] || this.URLS.support_contact.FR,
+                isExternal: true
+            });
+        }
+
+        return {
+            name: "assistance",
+            title: this.$translate.instant("common_menu_support_assistance"),
+            iconClass: "icon-assistance",
+            subLinks: assistanceMenu
+        };
+    }
+
+    getLanguageMenu () {
+        const currentLanguage = _.find(this.LANGUAGES.available, { key: this.translateService.getUserLocale() });
+
+        return {
+            name: "languages",
+            label: _(currentLanguage).get("name"),
+            title: _(currentLanguage).get("key").replace("_", "-"),
+            subLinks: _.map(this.LANGUAGES.available, (lang) => ({
+                title: lang.name,
+                click: function () {
+                    localStorage["univers-selected-language"] = lang.key;
+                    window.location.reload();
                 },
-                account: {
-                    label: this.$translate.instant("common_menu_account"),
+                lang: _.chain(lang.key).words().head().value()
+            }))
+        };
+    }
+
+    getUserMenu (currentUser) {
+        const getUserLinks = () => {
+            const isTarget = this.TARGET !== "EU";
```

Comment:
isEU or isTargetEU (and reverse it) would be more readable. At first, I thought it was if Target is specified...